corio.py

```diff
@@ -190,26 +192,26 @@ def setup_environment():
     assert ret, "Error while Mounting NFS directory"
 
 
-def log_status(parsed_input: dict, corio_start_time: datetime.time, test_failed):
+def log_status(parsed_input: dict, corio_start_time: datetime.time, test_failed: str):
     """
     Log execution status into log file.
 
     :param parsed_input: Dict for all the input yaml files
     :param corio_start_time: Start time for main process
     :param test_failed: Reason for failure is any
     """
-    LOGGER.info("Logging current status to corio_status.log")
-    with open('corio_status.log', 'w') as status_file:
+    status_fpath = os.path.join(
+        os.getcwd(), "reports", f"corio_status_{datetime.now().strftime('%d_%m_%Y_%H_%M_%S')}.log")
```

Comment:
Use `DT_STRING` instead of datetime.now(), so both report and log will have similar name

Reply:
we cannot use datetime.now() as only one file should be created for status log and the same is being updated every half hour.